src/backend/.eslintrc.json

```diff
@@ -8,7 +8,8 @@
     {
       "files": ["*.js", "*.ts"],
       "rules": {
-        "no-underscore-dangle": "off"
+        "no-underscore-dangle": "off",
+        "no-bitwise": "off"
```

Comment:
You can get rid of the no-bitwise rule if we use the uuid package; see other comment.